### ISSUE TITLE
The question about the implement of learning_rate

### ISSUE THREAD

**Issue by wenhaozheng-nju** [codertimo/BERT-pytorch#17]
Nice implements!  However, I have a question about learning rate. The learning_rate schedule which from the origin Transformers is warm-up restart, but your implement just simple decay. Could you implement it in your BERT code?

**Comment by codertimo** [comment]
@wenhaozheng-nju great point, I'll implement is ASAP. Thank you 👍 
Please let me know if you have any more request!

**Comment by codertimo** [comment]
@wenhaozheng-nju if you can implement this functionality, can you make a code and pull request?
I have no experience on think kind of things :) please let me know your thought

**Comment by wenhaozheng-nju** [comment]
You could refer to this implement:

https://github.com/jadore801120/attention-is-all-you-need-pytorch/blob/20f355eb655bad40195ae302b9d8036716be9a23/transformer/Optim.py#L4
